.cbmc-batch/include/cipher_openssl.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#ifndef CIPHER_OPENSSL_H
+#define CIPHER_OPENSSL_H
+
+#include <aws/cryptosdk/private/cipher.h>
+
+/* The definitions in this file are directly copied from cipher_openssl.c, in order to make the internals of the data
```

Comment:
What will happen in the future if the definitions drift apart?

Reply:
Ideally we should generate this header automatically from `cipher_openssl.c` so it doesn't become outdated, but I didn't have time to work on that. There is a tool developed by another intern that would allow us to do this, but it needs a few tweaks to handle this particular use case.